src/main/java/com/radixdlt/crypto/ECMultiplicationScalar.java

```diff
@@ -0,0 +1,26 @@
+/*
+ *
+ *  * (C) Copyright 2020 Radix DLT Ltd
```

Comment:
Confused copyright notice here (note the double asterisks)

Reply:
well spotted! I must have messed something up, thx!